### PR TITLE
Add main title and edit bar #2

### PR DESCRIPTION
**Pull request by hamzah-bahar**
### Main Title: 
- Three sizes: 
    1. Big screens 
image

    2. Medium screens
image

    3. Small screens
image

---

### Edit Bar: 
- Three screens as well: 
    *Big 
image

    *Medium
image
 
    *Small
image

---

image

image

image

### CODE REVIEW

**Comment by hamzah-bahar** [comment]
One small note: 
In style.css and media.css, the code under the comment (Right side) is for the left side and vice versa. 
My bad 😊 It is right from computer perspective though 😂

**Comment by Jood80** [comment]
neat description:bowtie: Just came up to my mind; you can replace this # of pictures with a gif for the whole screen while you are changing its size.
Also for the colors, try having a better contrast so that it didn't impact you accessibility :ok_hand: 
  
image

File: style.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
...4L360 27.3v25.4zm0-39.16v4.52l-2.47-1.43c.77-1.07 1.6-2.1 2.47-3.09zm0 52.92c-.87-.99-1.7-2.02-2.47-3.1l2.47-1.42v4.52zm0-16.07V29.61l-5.5-3.18a37.91 37.91 0 0 0 0 27.14l5.5-3.18zM62.42 360h2.16l3.11-6.78-4.85-3.52-7.68 7.08a49.83 49.83 0 0 0-4.98-2.54l1.21-10.38-5.7-1.85-5.12 9.11a49.9 49.9 0 0 0-5.52-.87L33 340h-6l-2.05 10.25c-1.85.19-3.7.48-5.52.87l-5.12-9.11-5.7 1.85 1.21 10.38c-1.71.75-3.37 1.6-4.98 2.54L0 352.32v5.17-2.5l4.62 4.26a47.84 47.84 0 0 1 7.33-3.74l-1.2-10.24 2.66-.86 5.05 8.99a47.91 47.91 0 0 1 8.12-1.28L28.6 342h2.8l2.02 10.12c2.78.2 5.5.63 8.12 1.28l5.05-9 2.66.87-1.2 10.24c2.54 1.04 5 2.29 7.33 3.74l7.58-7 2.26 1.65-2.8 6.1zM360 244.51l-1.44-.2-.8 1.38 2 2.54.24.17v-3.89zm0 14.45l-4-.4-3.16.66-.28 1.58 2.75 1.7 4.69 1.2v-4.74zm0 13.33l-4.7 1.2-2.74 1.71.28 1.58 3.16.66 4-.4v-4.75zm0 15.31l-.24.17-2 2.54.8 1.38 1.44-.2v-3.89zm0 5.76l-2.57.37-2-3.46 3.33-4.23 1.24-.85v8.17zm0-14.31l-3.65.36-5.27-1.1-.7-3.94 4.58-2.84 5.04-1.3v8.82zm0-13.28l-5.04-1.3-4.58-2.84.7-3.93 5.27-1.1 3.65.35v8.82zm0-14.96l-1.24-.85-3.33-4.23 2-3.46 2.57.37v8.17zm0 101.5V360h-4.58l-3.11-6.78 4.85-3.52 2.84 2.62v-.01zm0 2.67l-2.96-2.73-2.26 1.65 2.8 6.1H360v-5.02z'%3E%3C/path%3E%3C/svg%3E");
```
Comment (Jood80) [inline style.css]: hmm, Inline SVG is not abad Idea, however it makes your css file abit hard to follow,
have you considered separating it in an .svg file and call it back here?

Comment (hamzah-bahar) [reply style.css]: On it.

**Comment by hamzah-bahar** [comment]
lighthouse

* No accessibility issues anymore

**Comment by hamzah-bahar** [comment]
Inline svg was converted into `background.svg` file as requested.

**Review by Rahaf-96 — APPROVED** [review]

**Review by Jood80 — APPROVED** [review]

**Review by saharAdem — APPROVED** [review]
